### ISSUE TITLE
Build error using scss

### ISSUE THREAD

**Issue by VictorLessa** [team-innovation/vue-sfc-rollup#57]
Error: Error: Can't find stylesheet to import.
  ╷
1 │ @import "../../styles/_custom-variables/libs";
  │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ╵
  src\lib-components\forms\Spinner\VueNumberInputSpinner.vue 1:9  root stylesheet
    at Object.<anonymous> (D:\Documents\ks-components\node_modules\rollup-plugin-vue\dist\rollup-plugin-vue.js:273:35)
    at Generator.next (<anonymous>)
    at fulfilled (D:\Documents\ks-components\node_modules\rollup-plugin-vue\dist\rollup-plugin-vue.js:30:58)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ks-components@1.0.0 build: `cross-env NODE_ENV=production rollup --config build/rollup.config.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ks-components@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Victor\AppData\Roaming\npm-cache\_logs\2020-11-04T22_26_24_281Z-debug.log

I'm using scss, in development works, in production of this error

**Comment by mgdodge** [comment]
Out of the box, vue-sfc-rollup should inline the css, so if you are keeping it external, this isn't a totally vanilla config. Please provide a sample repo for this. It seems that you are possibly using external css/scss files, and when compiling they remain external. Since not all files in the 'src' directory become part of the published npm module, this would likely cause the error above.

**Comment by SumNeuron** [comment]
This is now an issue with Vuetify. They changed something so that my older rolled up components no longer maintain their styles when imported elsewhere. Please help

**Comment by mgdodge** [comment]
Is this still an issue? Do you have a sample repo? Looking to close this issue as stale.

**Comment by SvenBudak** [comment]
There is no sample repo required. just create a totally new lib with this cli and change in sample-component `<style scoped>` to `<style scoped lang="scss">` and you get the first errors.

**Comment by toniop99** [comment]
Is there any progress on this topic?

Getting this error when adding `<style scoped lang="scss">`
![Captura de pantalla 2021-07-12 a las 12 21 42](https://user-images.githubusercontent.com/35426566/125272144-ce497c80-e30b-11eb-8808-488e4e6bec3b.png)

And after install `sass-loader` this error appears, that seems to be related with the webpack configuration or maybe with some versioning conflict between `webpack` and` sass-loader.` **idk**
>  INFO  Starting development server...
 ERROR  Error: Rule can only have one resource source (provided resource and test + include + exclude) in {
  "exclude": [
    null
  ],
  "use": [
    {
      "loader": "/Users/chapikas/dev/npm-packages/vue/a-test-chapikas/node_modules/@vue/cli-plugin-babel/node_modules/cache-loader/dist/cjs.js",
      "options": {
        "cacheDirectory": "/Users/chapikas/dev/npm-packages/vue/a-test-chapikas/node_modules/.cache/babel-loader",
        "cacheIdentifier": "0fbb47ea"
      },
      "ident": "clonedRuleSet-38.use[0]"
    },
    {
      "loader": "/Users/chapikas/dev/npm-packages/vue/a-test-chapikas/node_modules/babel-loader/lib/index.js",
      "options": "undefined",
      "ident": "undefined"
    }
  ]
}
Error: Rule can only have one resource source (provided resource and test + include + exclude) in {
  "exclude": [
    null
  ],
  "use": [
    {
      "loader": "/Users/chapikas/dev/npm-packages/vue/a-test-chapikas/node_modules/@vue/cli-plugin-babel/node_modules/cache-loader/dist/cjs.js",
      "options": {
        "cacheDirectory": "/Users/chapikas/dev/npm-packages/vue/a-test-chapikas/node_modules/.cache/babel-loader",
        "cacheIdentifier": "0fbb47ea"
      },
      "ident": "clonedRuleSet-38.use[0]"
    },
    {
      "loader": "/Users/chapikas/dev/npm-packages/vue/a-test-chapikas/node_modules/babel-loader/lib/index.js",
      "options": "undefined",
      "ident": "undefined"
    }
  ]
}

**Comment by mgdodge** [comment]
It's not at all unreasonable to ask for a sample repo that reproduces the issue. I just ran the `sfc-init` command, chose to create a new library, and selected JavaScript. Once scaffolded, I ran `npm install` followed by `npm run build` with both plain CSS and SCSS, and both of them worked flawlessly. So without a sample repo, I find myself saying "works on my machine" which nobody likes. Without a sample repo that can demonstrate the issue, I can't look any deeper with confidence I can duplicate your issue.

That being said, I did notice that if I tried running the `npm run serve` script without installing `sass-loader` it warns me that it's missing. That is to be expected, and the vue cli does a great job informing me of the issue. When I run `npm install --save-dev sass-loader`, npm installs version 12, which is not compatible with webpack 4 (the default for the vue cli) and thus fails. If I run `npm install --save-dev sass-loader@10` it installs version 10, the last version compatible with webpack 4. Once that is in place, all things work fine, highlighting why checking versions of dependencies is an important troubleshooting step.

TL;DR is that I am closing this issue, since all my tests indicate that vue-sfc-rollup itself is working just fine. Without further clarification, it seems that the types of issues mentioned here come from an incomplete webpack setup or incompatible webpack loaders. Keep in mind that vue-sfc-rollup is not supposed to set up any specific CSS preprocessors or webpack configurations. It is intentionally very limited in its scope. SCSS is definitely widely used, but `sass-loader` is a webpack loader, and is thus outside the rollup configuration this cli tool is concerned with. Based on what I am seeing, this is not an issue with vue-sfc-rollup, but with additional configuration beyond its intended minimal output.

**Comment by TNortnern** [comment]
Has anyone been able to resolve this?

**Comment by ehsan-shv** [comment]
> Has anyone been able to resolve this?

Just add these two devDependencies to package.json:
```
    "sass": "^1.26.5",
    "sass-loader": "^8.0.2"
```
